__tests__/app.test.js

```diff
@@ -138,21 +138,55 @@ describe("/api/articles", () => {
                 });         
         });
     });
-    test("GET: 200 - articles are ordered by created_at in descending order", ()=>{
+    test("GET: 200 - articles are ordered by created_at in descending order by default", ()=>{
         return request(app)
         .get("/api/articles?sort_by=created_at")
         .expect(200)
         .then(({body}) => {
             expect(body.articles).toBeSortedBy("created_at", { descending: true})
         })
     });
-    test('GET: 200 - articles can be ordered ascending', () => {
+    test("GET: 200 - articles can be ordered ascending", () => {
         return request(app)
           .get('/api/articles?order=asc')
           .expect(200)
           .then(({ body }) => {
             expect(body.articles).toBeSortedBy('created_at');
```

Comment:
Is it also possible to sort these articles by another column besides created_at? We should be testing for that.